releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 7.2.0 - May 2, 2024
+### Enhancements
+* Added support for .NET 8 as a target framework. [#530](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/530)
+* Removed all usage of reflection for JSON parsing, allowing native AOT applications to use JSON key-values from Azure App Configuration with the .NET provider. [#537](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/537)
```

Comment:
How about this:
"Removed reflection-based JSON serialization to enable native AOT applications to utilize JSON key-values from Azure App Configuration."